src/dynamic_boundary_conditions/sea_level_rise_data.py

```diff
@@ -1,95 +1,173 @@
-# -*- coding: utf-8 -*-
-"""
-Created on Tue Oct 18 16:30:21 2022.
+import logging
+import pathlib
+from typing import Tuple, Union
 
-@author: alo78
-         xander.cai@pg.canterbury.ac.nz
-
-Used to get the sea level rise predictions (several predictions options to choose from)
-for site closest to the chosen latitude & longitude (Anywhere on the NZ Coastline) point
-from the csv's downloaded from: https://searise.takiwa.co/.
```

Comment:
Some top-level file docstrings are helpful, and are lacking in our project. I don't think it should be removed

Reply:
So far, I've been writing top-level file docstrings. However, since these two scripts are currently not performing exactly as intended, I'm considering adding these docstrings at the final stage, once everything is clarified.

Reply:
So far, I've been writing top-level file docstrings. However, since these two scripts are currently not performing exactly as intended, I'm considering adding these docstrings at the final stage, once everything is clarified.